spec/features/import_tp_call_data_spec.rb

```diff
@@ -5,10 +5,14 @@
 RSpec.feature 'Importing twilio call data', vcr: { cassette_name: 'twilio_single_page_of_data' } do
   let(:start_date) { Date.new(2016, 4, 11) }
   let(:end_date) { Date.new(2016, 4, 12) }
-  let(:tp_config) { double(:config, tp: { user_name: 'researchuploads@pensionwise.gob.uk', password: '1234' }) }
-
-  before do
-    allow(Rails.configuration).to receive(:x).and_return(tp_config)
+  let(:config) do
+    ActiveSupport::OrderedOptions[
```

Comment:
TIL `ActiveSupport::OrderedOptions`.

I would usually use an `OpenStruct` for the same purpose.